build.sbt

```diff
@@ -573,6 +573,7 @@ lazy val `fundamentals-bio` = project.in(file("fundamentals/fundamentals-bio"))
       "org.typelevel" %% "cats-core" % V.cats % Optional,
       "org.typelevel" %% "cats-effect" % V.cats_effect % Optional,
       "dev.zio" %% "zio" % V.zio % Optional,
+      "dev.zio" %% "zio-interop-cats" % V.zio_interop_cats % Optional,
```

Comment:
Could you add this just to `distage-framework`, not everywhere?

Reply:
Yup, a few sec, please)

Reply:
also, fix warnings while creating timer
```
 make[Timer[Task]].from(Timer[Task])
```